corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -173,7 +173,7 @@ hqDefine('cloudcare/js/backbone/cases.js', function () {
             var self = this,
                 text = self.lookupField(col),
                 td = $("<td/>");
-            if (text) {
+            if (text || text === 0) {
```

Comment:
not knowing exactly what's expected in text, would `text !== null` be more appropriate?